From 817fa1e015a93795acd3e20535f824fad0a314a0 Mon Sep 17 00:00:00 2001
From: Hao Bui <hao.bui.yg@renesas.com>
Date: Wed, 22 Apr 2020 11:32:49 +0700
Subject: [PATCH 369/445] ARM: dts: r8a7744: Add list of VSP phandles for DU

This patch adds phandles which identify the VSP instances that serves
the DU channel.

Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
Signed-off-by: Loc Vu <loc.vu.zn@renesas.com>
---
 arch/arm/boot/dts/r8a7744.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7744.dtsi b/arch/arm/boot/dts/r8a7744.dtsi
index 225ee46..b4b98ca 100644
--- a/arch/arm/boot/dts/r8a7744.dtsi
+++ b/arch/arm/boot/dts/r8a7744.dtsi
@@ -1665,6 +1665,8 @@
 			clock-names = "du.0", "du.1", "lvds.0";
 			status = "disabled";
 
+			vsps = <&vspd0 &vspd1>;
+
 			ports {
 				#address-cells = <1>;
 				#size-cells = <0>;
-- 
2.7.4

